sdk/tests/conformance2/textures/misc/copy-texture-image-webgl-specific.html

```diff
@@ -174,6 +174,140 @@
     gl.deleteTexture(texture[1]);
 };
 
+function copytexsubimage3d_test(xx, yy, copyWidth, copyHeight) {
+    var texture = [];
+    texture[0] = gl.createTexture();
+    texture[1] = gl.createTexture();
+    var layer = 0;
+    var width = 2;
+    var height = 2;
+    var depth = 2;
+    var width2 = 4;
+    var height2 = 4;
+    var xoffset = 0;
+    var yoffset = 0;
+    var uint = new Uint8Array(width * height * 4);
+    for (var i = 0; i < width * height * 4; i++) {
```

Comment:
To reduce code duplication this could just be:
  for (var i = 0; i < uint.length; i++) {
    ...